lib/routes/jin10/category.ts

```diff
@@ -0,0 +1,225 @@
+| 疫情           | 32   |
+| 疫苗动态       | 73   |
+| 确诊数据       | 74   |
+| 新冠药物       | 113  |
+| 债券           | 33   |
+| 政策           | 34   |
+| 中国           | 75   |
+| 美国           | 76   |
+| 欧盟           | 77   |
+| 日本           | 78   |
+| 贸易、关税     | 79   |
+| 碳中和         | 80   |
+| 中国香港       | 81   |
+| 英国           | 120  |
+| 房地产动态     | 156  |
+| 经济数据       | 35   |
+| 中国           | 82   |
+| 美国           | 83   |
+| 欧盟           | 84   |
+| 日本           | 85   |
+| 公司           | 37   |
+| 特斯拉         | 86   |
+| 苹果           | 90   |
+| 独角兽         | 91   |
+| 谷歌           | 92   |
+| 华为           | 93   |
+| 阿里巴巴       | 94   |
+| 小米           | 95   |
+| 字节跳动       | 116  |
+| 腾讯           | 117  |
+| 微软           | 118  |
+| 百度           | 119  |
+| 美团           | 162  |
+| 滴滴           | 163  |
+| 中国恒大       | 164  |
+| 灾害事故       | 38   |
+| 地震           | 96   |
+| 爆炸           | 97   |
+| 海啸           | 98   |
+| 寒潮           | 99   |
+| 洪涝           | 100  |
+| 火灾           | 101  |
+| 矿难           | 102  |
+| 枪击案         | 103  |
+`,
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['jin10.com/'],
+            target: '',
+        },
+    ],
+    name: '外汇',
+    maintainers: ['laampui'],
+    handler,
+    url: 'jin10.com/',
+};
+
+async function handler(ctx) {
+    const id = ctx.req.param('id');
+    const data = await cache.tryGet(
+        'jin10:aa:${category}',
+        async () => {
+            const { data: response } = await got('https://4a735ea38f8146198dc205d2e2d1bd28.z3c.jin10.com/flash', {
+                headers: {
+                    'x-app-id': 'bVBF4FyRTn5NJF5n',
+                    'x-version': '1.0',
+                },
+                searchParams: {
+                    channel: '-8200',
+                    vip: '1',
+                    classify: `[${id}]`,
+                },
+            });
+            return response.data.filter((item) => item.type !== 1);
+        },
+        config.cache.routeExpire,
+        false
+    );
+
+    const item = data.map((item) => {
+        const titleMatch = item.data.content.match(/^【(.*?)】/);
+        let title;
+        let content = item.data.content;
+        if (titleMatch) {
+            title = titleMatch[1];
+            content = content.replace(titleMatch[0], '');
+        } else {
+            title = item.data.vip_title || item.data.content;
+        }
+
+        return {
+            title,
+            description: art(path.join(__dirname, 'templates/description.art'), {
+                content,
+                pic: item.data.pic,
+            }),
+            pubDate: timezone(parseDate(item.time), 8),
+            link: item.data.link,
+            guid: `jin10:category:${item.id}`,
+            important: item.important,
```

Comment:
Useless property `important` as it's not listed in https://github.com/DIYgod/RSSHub/blob/f0854c7fa8e45f1d4f2457d72d1a2645db1fb5f2/lib/types.ts#L32